### PR TITLE
Add PMD Rules logging-java & imports

### PR DESCRIPTION
**Pull request by vanniktech**
As promised some more PMD rules.

### CODE REVIEW

File: code_quality_tools/ruleset.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    <description>Custom ruleset for Android application</description>
+
+    <exclude-pattern>.*/R.java</exclude-pattern>
+    <exclude-pattern>.*/gen/.*</exclude-pattern>
+
+    <rule ref="rulesets/java/imports.xml">
+        <exclude name="TooManyStaticImports"/>
+    </rule>
+    <rule ref="rulesets/java/unusedcode.xml"/>
+    <rule ref="rulesets/java/logging-java.xml"/>
```
Comment (artem-zinnatullin) [inline code_quality_tools/ruleset.xml]: Do you think it's useful for app that uses Timber? I'm afraid PMD won't recognize it at all.

Comment (vanniktech) [inline code_quality_tools/ruleset.xml]: I'd keep that rule in there. You can see all of the subrules [here](https://pmd.github.io/pmd-5.4.1/pmd-java/rules/index.html#Java_Logging). They all seem logical to me.

Also Timber has some built in Lint checks that also do some checking that you should not use Log.v anymore etc.

Comment (artem-zinnatullin) [inline code_quality_tools/ruleset.xml]: Now I see that it also checks for `e.printStackTrace`, ok!

**Comment by artem-zinnatullin** [comment]
:+1: thanks!

**Pull request merged**